app/locale/en_US/Mage_Cms.csv

```diff
@@ -27,8 +27,10 @@
 "CMS Static Block","CMS Static Block"
 "CMS Static Block Default Template","CMS Static Block Default Template"
 "Cannot create new directory.","Cannot create new directory."
+"Cannot delete page, it is used in "%s".","Cannot delete page, it is used in "%s"."
 "Cannot delete directory %s.","Cannot delete directory %s."
 "Cannot delete root directory %s.","Cannot delete root directory %s."
+"Cannot disable page, it is used in "%s".","Cannot disable page, it is used in "%s"."
```

Comment:
`"Cannot disable page, it is used in ""%s"".","Cannot disable page, it is used in ""%s""."`

Reply:
Is it necessary? Tested it and lgtm?

Reply:
Yes it works, but I think in docs of CSV files you need to escape them.

Reply:
AFAIK it is necessary to doubloe quote quotes in CSV

Reply:
Can you make pls a "suggestion"?